test/layers/basic.jl

```diff
@@ -28,6 +28,13 @@ import Flux: activations
   end
 
   @testset "Dense" begin
+    @testset "constructors" begin
+      @test size(Dense(10, 100).W) == (100, 10)
+      @test Dense(rand(100,10), rand(10)).σ == identity
+
+      @test_throws MethodError Dense(10, 10.5)
```

Comment:
lets also even test amost hitting the number constructor
```suggestion
      @test_throws MethodError Dense(10, 10.5)
      @test_throws MethodError Dense(10, 10.5; σ=tanh)
```